.github/ISSUE_TEMPLATE/v1-bug-report.md

```diff
@@ -59,3 +55,9 @@ We'd love to have more contributors on this project! If the fix for this bug is
 ```
 # paste `go env` output in here
 ```
+
+## Run `go list -u -m all` and paste its output here
+
+```
+# paste `go list -u -m all` output in here
```

Comment:
```
~ $ go list -u -m all
warning: pattern "all" matched no module dependencies
```

What is this supposed to do?

Reply:
Ok, we have to be more explicit where this needs to be run. This also can be pretty big, are we sure we want everything?
What are we trying to answer here?

Reply:
> What is this supposed to do?

I added that into my pull request description

```
❯ go list -u -m all
go: finding github.com/cpuguy83/go-md2man/v2 v2.0.0
go: finding gopkg.in/check.v1 latest
go: finding gopkg.in/yaml.v2 v2.2.7
github.com/urfave/cli/v2
github.com/BurntSushi/toml v0.3.1
github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d [v2.0.0]
github.com/pmezard/go-difflib v1.0.0
github.com/russross/blackfriday/v2 v2.0.1
github.com/shurcooL/sanitized_anchor_name v1.0.0
gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405 [v1.0.0-20190902080502-41f04d3bba15]
gopkg.in/yaml.v2 v2.2.2 [v2.2.7]
```

did you run it inside of a golang github repo? what go version are you using?

> This also can be pretty big, are we sure we want everything?

Yes

> What are we trying to answer here?

The versions of your dependencies. Most critically the version of urfave/cli

Reply:
I think we can pipe this to ```grep urfave/cli``` to just focus on the version of our package.